.github/workflows/checks.yaml

```
@@ -0,0 +1,56 @@
+      source_branch: ${{ github.event.pull_request.head.ref || github.ref }}
+    secrets: inherit
+  zizmor:
+    name: Zizmor
+    uses: ./.github/workflows/checks-zizmor.yaml
+    with:
+      source_repo: ${{ github.event.pull_request.head.repo.full_name || github.repository }}
+      source_branch: ${{ github.event.pull_request.head.ref || github.ref }}
+    secrets: inherit
+  performance:
+    name: Performance
+    uses: ./.github/workflows/checks-performance.yaml
```

actionlint (1.7.8)

could not read reusable workflow file for "./.github/workflows/checks-performance.yaml": open /home/jailuser/git/.github/workflows/checks-performance.yaml: no such file or directory

(workflow-call)